File: {{cookiecutter.project_slug}}/src/main/java/{{ cookiecutter.package_path }}/command/ZeroGyroCommand.java
package {{cookiecutter.package_name}}.command;

import edu.wpi.first.wpilibj.command.InstantCommand;
import {{cookiecutter.package_name}}.Robot;
import {{cookiecutter.package_name}}.subsystem.DriveSubsystem;

public final class ZeroGyroCommand extends InstantCommand {

  private final static DriveSubsystem swerve = Robot.DRIVE;

  public ZeroGyroCommand() {
    requires(swerve);
  }

  @Override
  protected void initialize() {
    swerve.zeroGyro();
  }
}

File: {{cookiecutter.project_slug}}/src/main/kotlin/{{ cookiecutter.package_path }}/command/ZeroGyroCommand.kt
package {{cookiecutter.package_name}}.command

import edu.wpi.first.wpilibj.command.InstantCommand
import {{cookiecutter.package_name}}.Robot

class ZeroGyroCommand : InstantCommand() {
    private val swerve = Robot.DRIVE

    init {
        requires(swerve)
    }

    override fun initialize() = swerve.resetGyro()
}